bundle/Resources/config/services.yml

```diff
@@ -6,7 +6,6 @@ parameters:
     ezpublish.default_router.legacy_aware_routes: ['_ezpublishLegacyTreeMenu', 'ezpublish_rest_', '_ezpublishPreviewContent', '_wdt', '_profiler', '_assetic']
 
     # Core overrides
-    router.class: eZ\Bundle\EzPublishLegacyBundle\Routing\DefaultRouter
     ezpublish.security.login_listener.class: eZ\Bundle\EzPublishLegacyBundle\Security\SecurityListener
     security.authentication.listener.rememberme.class: eZ\Bundle\EzPublishLegacyBundle\Security\RememberMeListener
```

Comment:
Are you suggesting another PR just for the parameter removal?

Not quite sure I follow.  As the compiler pass now sets the class, this parameter is not used anyway.

Let me know what you want done.